### ISSUE TITLE
Accessibility: Boolean custom field icon lacks a text alternative

### ISSUE THREAD

**Issue by 51bJwFuhUC2fg2yvZ8zL4815sEvsml4s7swbpcz** [bitwarden/mobile#1987]
### Steps To Reproduce

1. Use a screen reader (VoiceOver on iOS/Talkback on Android)
2. Go to an entry with a custom boolean field
3. Navigate to the custom boolean field

### Expected Result

A screen reader show announce the boolean as either checked/unchecked, on/off, true/false, or similar

### Actual Result

The custom character/icon is passed to screen readers as the only value. As this is a custom character, it's not announced at all - VoiceOver / Talkback are silent when reaching the character.

### Screenshots or Videos

iOS/VoiceOver screenshot - once focus is on the custom checked checkbox icon, note the voice output (bottom of the screen) is just showing the unrecognised custom character. Nothing is announced.

![iOS screenshot](https://user-images.githubusercontent.com/895831/178105830-d65b2a07-2bf5-4719-96b1-8274475242fc.png)

---

Android/Talkback screenshot - once focus is on the custom checked checkbox icon, note the voice output (bottom of the screen) is just showing the unrecognised custom character. Nothing is announced. 

![image0](https://user-images.githubusercontent.com/895831/178105929-64c9f618-ba13-4d83-a7e1-b7ad86a3a6b0.jpg)

### Additional Context

Note that in the extension, and in the desktop app, the boolean checkbox is announced as `true` or `false`.

Example using desktop app and NVDA/Windows
![desktop app and NVDA speech output, showing it announces the unchecked checkbox as "false"](https://user-images.githubusercontent.com/895831/178105974-b46de3fc-cf53-4f00-8ada-3122ed6ea249.png)

Example using chrome extension and NVDA/Windows
![chrome extension and NVDA speech output, showing it announces the unchecked checkbox as "false"](https://user-images.githubusercontent.com/895831/178105992-b5526fbf-1b35-4ce1-976a-f07f2a318e91.png)

This is done with a visually-hidden, but exposed to AT, `<span>` element that contains the actual value of the field

![desktop app with developer tools open, showing the hidden span after the checkbox icon containing the value of the field](https://user-images.githubusercontent.com/895831/178106024-efb383ff-e404-4dc4-9579-e953316cc8b8.png)

### Operating System

Android, iOS

### Operating System Version

Latest

### Device

iPhone SE (2020), Motorola moto g8

### Build Version

2022.6.0 (1859) iOS / 2022.6.1 (4795) Android

### Beta

- [ ] Using a pre-release version of the application.

**Comment by closebot-bw** [comment]
This repo has been archived. To report an issue you're experiencing in the new native apps, visit the [iOS](https://github.com/bitwarden/ios) / [Android](https://github.com/bitwarden/android) repo.
